src/components/Table/nestedCards.tsx

```diff
@@ -42,22 +50,14 @@ export class NestedCards {
     this.openCards.pop();
   }
 
-  addSpacerBetweenChildren() {
-    const openCard = this.openCards[this.openCards.length - 1];
-    // If we're between two top-level cards, there is no open card, so fallback on topLevelSpacerPx
-    const height = openCard?.spacerPx || this.style.nestedCards!.topLevelSpacerPx;
```

Comment:
Cool, yeah let's remove the `NestedCardStyle.spacerPx` then.

Reply:
I believe I did! Will double check.

- [ ] Validate that `NestedCardStyle.spacerPx` is removed.

Reply:
I've confirmed that it's removed and also removed any lingering usages.